src/ansible_navigator/app.py

```diff
@@ -83,6 +85,19 @@ def app(self) -> AppPublic:
             )
         raise AttributeError("app passed without args initialized")
 
+    def no_interactive_mode(self, interaction: Interaction, app: AppPublic) -> None:
+        # pylint: disable=unused-argument
+        """Warn the user interactive mode is not supported"""
```

Comment:
Would this be more precise?
```suggestion
        """Show a warning notification that the user interactive mode is not supported."""
```